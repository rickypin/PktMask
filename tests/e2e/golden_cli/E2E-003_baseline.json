{
  "test_id": "E2E-003",
  "name": "Mask Only",
  "input_file": "tests/data/tls/tls_1_2-2.pcap",
  "input_hash": "baf39e40187111119c0de19358b3c944a07489806231fd24e6ab922b678b3ded",
  "cli_args": [
    "--mask"
  ],
  "output_hash": "f638af03eb9dfae58fbf5a8757c2c0b0947bfa2803687d9e18f2ec92ed3b5b03",
  "timestamp": "2025-10-09T21:31:09.354689",
  "version": "1.0-cli",
  "note": "Generated using CLI only - completely decoupled from API"
}