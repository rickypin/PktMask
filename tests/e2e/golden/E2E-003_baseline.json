{
  "test_id": "E2E-003",
  "name": "Mask Only",
  "input_file": "tests/data/tls/tls_1_2-2.pcap",
  "input_hash": "baf39e40187111119c0de19358b3c944a07489806231fd24e6ab922b678b3ded",
  "config": {
    "dedup": false,
    "anon": false,
    "mask": true
  },
  "output_hash": "f638af03eb9dfae58fbf5a8757c2c0b0947bfa2803687d9e18f2ec92ed3b5b03",
  "output_file_size": 2721,
  "packet_count": 14,
  "stats": {
    "packets_processed": 14,
    "packets_modified": 1,
    "duration_ms": 414.370059967041,
    "stages": [
      {
        "name": "Mask Payloads",
        "packets_processed": 14,
        "packets_modified": 1,
        "duration_ms": 9.16910171508789
      }
    ]
  },
  "timestamp": "2025-10-09T19:39:45.488472",
  "version": "1.0"
}